### DISCUSSION TITLE
Remove default "is:open" tag from Newcomers and maybe others

### DISCUSSION THREAD

**Discussion by MCVenor** [stride3d/stride#1937] [General]
Hi,

i would like to put forth a suggestion to remove the default "is:open " tag when someone clicks on a category that might very well interest them in topics that are "closed" or "other status".

Newcomers, View all discussions and Show+Tell are those categories, QnA maybe too.

But especially **Newcomers** should by default see that there are others that freely ask questions, see topics that might spark their curiosity or show a problem that they are about to post.

**Comment by VaclavElias** [comment]
Can you actually do it somewhere in the Settings?

**Reply by tebjan** [reply]
I haven't seen this setting anywhere, unfortunately. But we can remove the question/answer mode, not sure if that would solve it.

**Reply by VaclavElias** [reply]
Q&A should be ok, because one can see quickly that a question was answered. We just shouldn't close the discussion https://github.com/stride3d/stride/discussions?discussions_q=is%3Aclosed unless it has no value to be opened.

That means, anything what get answered or resolved, should stay open, because that is default view. Maybe we should open some of those closed?
